missilewars-plugin/src/main/java/de/butzlabben/missilewars/listener/GameListener.java

```diff
@@ -257,6 +239,7 @@ public void onRespawn(PlayerRespawnEvent event) {
             event.setRespawnLocation(team.getSpawn());
             getGame().sendGameItems(player, true);
             getGame().setPlayerAttributes(player);
+            getGame().getPlayer(player).getRandomGameEquipment().resetPlayerInterval();
```

Comment:
Maybe make this configurable? Some people maybe want to configure their game to be faster and have the time not be reset upon death.

Reply:
Currently, it's the start interval. If you down't reset the interval and die while the interval is at 0 (and you lose your items at this time), you have longer to wait as wished.

```
equipment_interval:
  start: 10
  intervals_by_team_amount:
    "1": 15
    "2": 20
    "4": 25
````